examples/incompressible_flow/3d_vessels/vessels.prm

```diff
@@ -0,0 +1,140 @@
+# Listing of Parameters
+# ---------------------
+
+
+# --------------------------------------------------
+# Simulation Control
+#---------------------------------------------------
+subsection simulation control
+  set method                  = bdf1
+  set time step               = 0.0001
+  set time end                = 10
+  set output name             = vessels
+  set output path             = ./output/
+  set output frequency        = 10
+end
+
+#---------------------------------------------------
+# FEM
+#---------------------------------------------------
+subsection FEM
+    set velocity order            = 1
+    set pressure order            = 1
+    set tracer order              = 1
+    set qmapping all              = false
```

Comment:
qmapping is set to false by default, do you really want to keep that option visible here and talk about it or not?

Reply:
I wouldn't want to talk about it